content/java/concepts/errors/terms/stringindexoutofboundsexception/stringindexoutofboundsexception.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'StringIndexOutOfBoundsException'
+Description: 'Occurs when a String method tries to use an index that is either negative or greater than the size of the string.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Errors'
+  - 'Error Handling'
+  - 'Exceptions'
+  - 'Debugging'
+  - 'Strings'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `StringIndexOutOfBoundsException` is an [exception](https://www.codecademy.com/resources/docs/general/error) that occurs when a [`String`](https://www.codecademy.com/resources/docs/java/strings) method tries to use an index that is either negative or greater than the size of the `String`.
+
+## Example
+
+The following example creates a `StingIndexOutOfBoundsException`:
```

Comment:
```suggestion
The following example creates a `StringIndexOutOfBoundsException`:
```

![image of Sting of English band The Police](https://user-images.githubusercontent.com/23144845/184493902-41451cff-a29c-462b-8196-ce24ffb6fbef.png)